Makefile

```diff
@@ -124,14 +126,9 @@ helm:
 tidy:
 	@go mod tidy
 
-# ---------------------
-##@ Testing and Linting
-# ---------------------
-
-## Runs code-generators, checks for clean directory and lints the source code.
-lint:
-	./mage test:lint
-.PHONY: lint
+# -----------
+##@ Testing
+# -----------
 
 ## Runs unittests.
 test-unit:
```

Comment:
Found the potential issue, `coverage` references `go-test` which runs against all the possible packages including integration which isn't viable. We can override Boilerplate's target by referencing the same name in our operator Makefile.

Ref: https://github.com/openshift/addon-operator/pull/506/files#diff-030411ad9aef71dbbacb50854684224cf38b2c799da1c964f9706bc8a8408a2aR17

```suggestion
go-test:
```

Reply:
The target has been fixed but I expect the unit test CI check to fail as the Prow config still references `test-unit` that'll be changed soon.